lib/linguist/heuristics.rb

```diff
@@ -491,5 +499,14 @@ def call(data)
         Language["XML"]
       end
     end
+
+    disambiguate ".user" do |data|
```

Comment:
No, actually, I'd leave `.user` out as a registered file extension altogether. Unless you can link to some *canonical* documentation where that extension is promulgated as official.